src/Microsoft.Health.Fhir.Azure/IntegrationDataStore/AzureBlobIntegrationDataStoreClient.cs

```diff
@@ -178,6 +178,13 @@ await _integrationStoreRetryExceptionPolicyFactory
                                 return result;
                             });
             }
+            catch (AggregateException ex) when (ex.InnerExceptions[0] is RequestFailedException)
+            {
+                var innerException = (RequestFailedException)ex.InnerExceptions[0];
+                _logger.LogInformation(innerException, "{Error}", innerException.Message);
+                Exception exception = HandleRequestFailedException(innerException, "Failed to get properties of blob {Url}");
```

Comment:
Missing string interpolation?
```suggestion
                Exception exception = HandleRequestFailedException(innerException, $"Failed to get properties of blob {Url}");
```

Reply:
Added interpolation